### DISCUSSION TITLE
Citation Key: how to add Zotero 'Item Type' value?

### DISCUSSION THREAD

**Discussion by slhenty** [retorquere/zotero-better-bibtex#2410] [Q&A] [answered]
I'm not seeing a way to add the Zotero 'Item Type' value to the citation key. Does anyone have an example of how to do this?

**Comment by retorquere** [comment]
You can't currently, but if you open a new issue, I can add that capability.

**Reply by mialuyao** [reply]
Has it been added currently?

**Reply by retorquere** [reply, marked as answer]
You can get the item key using `item(key)`
